base/deprecated.jl

```diff
@@ -2097,6 +2107,11 @@ end
     @deprecate chol!(x::Number, uplo) chol(x) false
 end
 
+# #24490
+# info() warn() and logging() are deprecated (see util.jl). When removing
+# these, the functions redirect() and _redirect() should also be removed, along
+# with log_info_to, log_warn_to and log_error_to.
```

Comment:
A small question: is it ok to put this note here, or should the parts of `util.jl` which are deprecated be explicitly moved? It's 200 lines of code or so which is why I left it in place for now.